cogs/logcog.py

```diff
@@ -0,0 +1,40 @@
+from discord.ext import commands
+import discord
+import json
+
+class LogCog(commands.Cog):
+    def __init__(self, bot):
+        self.bot = bot
+
+    @commands.command()
+    @commands.has_role("Bot Commander")
+    async def setlogchannel(self, ctx, channel):
+        channel_obj = discord.utils.get(ctx.guild.text_channels, name=channel)
+        if channel_obj == None:
+            await ctx.send("Invalid channel!")
+            return
+
+        with open("jsondata/logchannel.json", "r") as f:
```

Comment:
we currently have a directory called `data` and a directory called `jsondata` which essentially serve the same purpose for different cogs. Maybe we should centralise?